projects/ion/src/lib/indicator/indicator.component.spec.ts

```diff
@@ -135,11 +138,23 @@ describe('IonIndicatorComponent', () => {
     expect(getElementByTestId('value').textContent).toBe(
       valueNumber.toString()
     );
-    expect(getElementByTestId('secondValue').textContent).toBe(
+    expect(getElementByTestId('secondValue').textContent.trim()).toBe(
```

Comment:
Why a trim() is required now?

Reply:
if I put the `{{ secondValue }}` without any space for the `<p>` tag it doesn't need `.trim()`
![image](https://github.com/Brisanet/ion/assets/138119077/22c9e05d-8a09-447a-a52c-b0f85507a273)

but, as lint automatically formats it to look like this, `.textContent` returns the value with a space before and after
![image](https://github.com/Brisanet/ion/assets/138119077/58095243-a1ab-4629-a9a1-b873d5548061)

Reply:
Ahh got it, I think I had the same issue once. Thanks for explaining.